package.json

```diff
@@ -49,6 +49,7 @@
         "lint:check": "eslint **/*.{js,ts,tsx}",
         "lint:check:all": "yarn lint:check && npx lerna run --npm-client yarn lint:check",
         "lint:fix": "eslint **/*.{js,ts,tsx} --quiet --fix",
+        "unified-mac-sign": "node ./pipeline/scripts/mac-sign.js",
```

Comment:
nit: keep the list alphabetized?

Reply:
Good catch! I think I initially had it as mac-sign and forgot to update it!